docs/docs/cloud/how-tos/human_in_the_loop_breakpoint.md

```diff
@@ -36,6 +36,17 @@ In this how-to we use a simple ReAct style hosted graph (you can see the full co
     const thread = await client.threads.create();
     ```
 
+=== "CURL"
+
+    ```bash
+    curl --request POST \
+      --url <DEPLOYMENY_URL>/threads \
```

Comment:
maybe for consistency we can replace "whatever-your-deployment-url-is" with <DEPLOYMENT_URL>? also, i think there is a typo "<DEPLOYMENY_URL>"